content/workflows/dry-run.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: bt_wiki
+title: Dry Run Workflow Execution
+category: Workflows
+draft: false
+abstract: Explanation of running workflows without side effects
+weight: 550
+
+
+types_yaml_link: reference-types.html
+
+default_workflows_source_link: https://github.com/cloudify-cosmo/cloudify-plugins-common/blob/4.3/cloudify/plugins/workflows.py
+---
+
+{{% gsSummary %}}{{% /gsSummary %}}
+
+## Overview
+
+In a dry-run execution, you can execute a workflow so that the entire flow of the execution (all the operations that are executed in an actual run) is shown, but no actual code is executed and there are no side effects.
+
+A dry-run is useful in the process when you write complex blueprints with potentially long executions. The dry-run helps you to configure relationships between node types, and operations that depend on those relationships.
```

Comment:
"relationships between node types" > "relationships between node templates". You don't have relationships between node types.